src/screens/SurveyOverviewScreen.tsx

```diff
@@ -80,6 +101,15 @@ export const SurveyOverviewScreen = () => {
   return (
     <SafeAreaViewFlex>
       <SectionList
+        ListHeaderComponent={
+          <>
+            {!!data && (
+              <Wrapper>
+                <HtmlView html={data} />
+              </Wrapper>
+            )}
+          </>
+        }
```

Comment:
I suggest passing `null` here instead of having an empty fragment always for rendering, even if there is no `data`. We do not want any `ListHeaderComponent` for that case, right:

```
ListHeaderComponent={
  !!data ? (
    <Wrapper>
      <HtmlView html={data} />
    </Wrapper>
  ) : null
}
```

Reply:
Yeah, you're right. I think I missed that.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/d3e2bb5ccf4ca1da43e67f69877b7b68207df161

Reply:
Ow. I apologise. It was my carelessness. 🫣
https://github.com/smart-village-solutions/smart-village-app-app/commit/57301acb3d4274e87de430fd447912e8242b4342